bedms/attr_standardizer.py

```diff
@@ -28,13 +38,17 @@
     fetch_from_pephub,
     get_any_pep,
     load_from_huggingface,
+    hf_hub_download,
```

Comment:
Isnt this from the `huggingface_hub` package? Shouldnt we import directly there? Or is this some sort of shadowed wrapper?